src/porepy/numerics/fv/biot.py

```diff
@@ -386,7 +392,7 @@ def build_rhs_normals_single_dimension(dim):
         num_dir_subface = (
             bound_exclusion_mech.exclude_neu.shape[1]
             - bound_exclusion_mech.exclude_neu.shape[0]
-        ) * nd
+        )
```

Comment:
Is this a bugfix?

Reply:
Yes, bound_exlusioin.exlude_neu (and _dir, etc) changed from 1 per face to nd per face at some point (related to mpsa always getting BoundaryConditionVectorial).

Reply:
Yes, because bound_exclusion has changed from operating